src/main/java/hudson/plugins/pmd/PmdPublisher.java

```diff
@@ -25,14 +24,20 @@
  * @author Ulli Hafner
  */
 public class PmdPublisher extends HealthAwarePublisher {
+
     private static final long serialVersionUID = 6711252664481150129L;
```

Comment:
No never ever change a `serialVersionUID` unless you cannot deserialize a correct object from the older format.

An example of when to change:
* You add a `NonNull` field and do not provide a `readResolve` method that enforces the non-null when old versions are deserialize and you want to screw your users over in strange and obtuse ways.

In Jenkins you basically *never* want to change the value, hence why you should just start every `Serializable` with the value `1L` so that we know from the start what the value is. The only time to use a value that is not `1L` is when you shipped by mistake... then you use the value that was shipped and leave it alone thereafter.